### DISCUSSION TITLE
SharePoint PnP Modern Search v4 to Display results from a specific folder in a library

### DISCUSSION THREAD

**Discussion by RogerCP66** [microsoft-search/pnp-modern-search#3661] [Q&A] [answered]
Hello,

I am trying to setup PnP search on a site to return results from a specific folder in another site. When I connect the results display, I am able to get the search results working and showing results from the folder I want. I can’t seem to get the related search bar to work.

This is my query on the search results which is returning the correct number of pictures that are in the folder.

{searchTerms}Path:"https://tenant. sharepoint.com /sites /sitename/ libraryname"

(FileType:jpg OR FileType:jpeg OR FileType:png OR FileType:tif)

RootFolder:"foldername"

 on page 3 I select Use Input Query text >Dynamic Value> connect to source (PnP - SearchBox)> Pnp Search Box Properties (Search Query)

I then added the search box and under available connections selected connect to source (PnP Search Results Properties selected the library> then the third dropdown for the properties and am presented with 2 options (AvailableFieldsFromResults and AvailableFilters). Either selection does not change the results.

I can always get the results to show up from the correct folder but cannot get the search box to show me any results when I try to refine the search.

**Comment by kasperbolarsen** [comment]
Hey Roger,

You don't need to config anything on the search box, only on the result web part:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/5aa785af-581b-4f10-9fc4-774f67f8088c)

**Reply by RogerCP66** [reply]
Hey @kasperbolarsen not sure I understand. I have tried that but then I get no results returned at all.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/145495774/8455b5b3-0386-4183-b041-f4df6b78b1ae)

Results when searching for dog:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/145495774/0db98766-6d8a-4808-a30d-f9cc9e1d3bf8)

"Dog" photo in library:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/145495774/073cd36a-d192-4020-b7ac-ddf73ba8adf7)

**Comment by kasperbolarsen** [comment, marked as answer]
I think I can see the error, perhaps:
change {searchTerms}Path:"https://tenant/. sharepoint.com /sites /sitename/ libraryname" to 
{searchTerms} Path:"https://tenant/. sharepoint.com /sites /sitename/ libraryname"  (added a space before Path ;-) )

**Reply by RogerCP66** [reply]
Hey @kasperbolarsen  awesome catch. That worked, thank you for your help.

**Reply by kasperbolarsen** [reply]
You are welcome
